TreeDB/db/standalone_leaf_page_stable_resource_test.go

```
@@ -12,8 +14,627 @@ import (
+	writer.ConfigureLeafLog(LeafLogDirPath(dir), rewriteLeafLogLaneID, 0)
+	writer.SetTemplateCompression(templ.TemplateOnly, templ.Config{
+		MinSavingsBytes: 1, FingerprintK: 8, FingerprintW: 8,
+		MaxFingerprints: 32, MaxFPReads: 32, MaxCandidatesPerFP: 8, MaxTemplateFetch: 8,
+	}, provider)
+	database.SetLeafPageLog(writer)
+	t.Cleanup(func() {
+		_ = database.Close()
+		_ = writer.Close()
+	})
+
+	ptrs, resources, err := database.leafPageLog.(LeafPageStableBatchLog).AppendLeafPagesWithStableResources(pages)
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)